addons/medical_feedback/functions/fnc_handleGUIIndication.sqf

```diff
@@ -0,0 +1,63 @@
+#include "script_component.hpp"
+/*
+ * Author: 10Dozen
+ * Handles indication of the fractures, applied tourniquets and splints over Stance indicator.
+ * Draws an icon if there is at least 1 fracture/splint/tourniquet applied.
+ *
+ * Arguments:
+ * 0: Drop indicator <BOOL>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [false] call ace_medical_feedback_fnc_handleGUIIndication
+ *
+ * Public: No
+ */
+params [["_dropIndicator", false]];
+private _indicatorSlots = [
+    uiNamespace getVariable [QGVAR(stateIndicator1), controlNull],
+    uiNamespace getVariable [QGVAR(stateIndicator2), controlNull],
+    uiNamespace getVariable [QGVAR(stateIndicator3), controlNull]
+];
+
+// --- Removes any indication and exit
+if (_dropIndicator) exitWith {
+    {
+        _x ctrlSetText "";
+    } forEach _indicatorSlots;
+};
+
+// --- Tourniquets
+private _hasTourniquets = {_x > 0} count GET_TOURNIQUETS(ACE_player) > 0;
```

Comment:
Using `count` for "has" checks is not optimal, `findIf` stops iteration on first found element.
```suggestion
private _hasTourniquets = GET_TOURNIQUETS(ACE_player) findIf {_x > 0} != -1;
```